src/node.rs

```diff
@@ -265,13 +315,16 @@ impl PbftNode {
         }
     }
 
-    fn check_if_checkpoint_started(&mut self, msg: &PeerMessage) -> bool {
+    fn check_if_checkpoint_started(&mut self, msg: &PeerMessage, sender_id: &PeerId) -> bool {
         // Not ready to receive checkpoint yet; only acceptable in NotStarted
         if self.state.phase != PbftPhase::NotStarted {
-            self.msg_log.push_backlog(PeerMessage {
-                message_type: msg.message_type.clone(),
-                content: msg.content.clone(),
-            });
+            self.msg_log.push_backlog(
+                PeerMessage {
```

Comment:
Just opened https://github.com/hyperledger/sawtooth-core/pull/1905 to be able to do `msg.clone()`

Reply:
Updated this to take advantage of the `.clone()` method